src/misc/cache/_lv_cache_lru_rb.c

```diff
@@ -440,7 +440,8 @@ static lv_cache_reserve_cond_res_t reserve_cond_cb(lv_cache_t * cache, const voi
 
     uint32_t data_size = key ? lru->get_data_size_cb(key) : 0;
     if(data_size > lru->cache.max_size) {
-        LV_LOG_ERROR("data size (%" LV_PRIu32 ") is larger than max size (%" LV_PRIu32 ")", data_size, lru->cache.max_size);
+        LV_LOG_ERROR("data size (%" LV_PRIu32 ") is larger than max size (%" LV_PRIu32 ")", data_size,
```

Comment:
Use `%zu` to print `size_t` type without casting. 
See: https://stackoverflow.com/questions/2524611/how-can-one-print-a-size-t-variable-portably-using-the-printf-family

Reply:
> Use `%zu` to print `size_t` type without casting. See: https://stackoverflow.com/questions/2524611/how-can-one-print-a-size-t-variable-portably-using-the-printf-family

Did not know that this exist. Should have tried tried to google it..

Should I add a new `LV_PRIsize_t` define for it, oder just use the the *raw* `%zu` format string

Reply:
You can use `%zu` directly. It directly matches `size_t` and does not require adding macros for conversion.